src/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+
+import { Typography, Box } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CardItemInterface {
+  item: {
+    id: string;
+    imageSrc: string;
+    title: string;
+    text: string;
+  };
+  type: 'WhyUs' | 'Possibilities';
+}
+
+function CardItem({ item, type }: CardItemInterface) {
+  const { t } = useTranslation();
+
+  return (
+    <Box>
+      {type === 'Possibilities' && (
```

Comment:
this is the incorrect implementation of this component, previously i add a comment and on the call explained how this component must be looked at, this part must be moved to another component and be rendered here by type